modules/auto-delete/events/voiceStateUpdate.js

```diff
@@ -0,0 +1,18 @@
+module.exports.run = async function (client, oldState) {
+    if (!client.botReadyAt) return;
+
+    const voiceChannels = client.configurations['auto-delete']['voice-channels'];
+
+    const channelConfigEntry = {...voiceChannels.find((vc) => oldState.channelId === vc.channelID)};
```

Comment:
why do we need to clone this object? Using .find should be enough, this only creates an empty object if nothing was found. I highly suggest changing this behavior and returning if `channelConfigEntry` is null

Reply:
Fixed it